### ISSUE TITLE
syncthing-gtk never connects to syncthing, throws TypeError

### ISSUE THREAD

**Issue by codl** [syncthing/syncthing-gtk#432]
![2018-02-27t12-59-22-ooo0](https://user-images.githubusercontent.com/315139/36727580-1278ed00-1bbe-11e8-9466-37c98c2c9d77.png)

```
$ syncthing-gtk
I StatusIcon    Using backend StatusIconGTK3 (primary)
(syncthing-gtk:4940): Gtk-WARNING **: Invalid icon size 24
(syncthing-gtk:4940): Gdk-CRITICAL **: gdk_window_thaw_toplevel_updates: assertion 'window->update_and_descendants_freeze_count > 0' failed
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/syncthing_gtk/daemon.py", line 398, in _request_config
    RESTRequest(self, "system/config", self._syncthing_cb_config, self._syncthing_cb_config_error).start()
  File "/usr/lib/python2.7/site-packages/syncthing_gtk/daemon.py", line 1045, in __init__
    self.connect("event", self._socket_event)
TypeError: argument connectable: Expected Gio.SocketConnectable, but got unicode
```

i've tried to clear out `$XDG_CONFIG/syncthing-gtk` but to no avail

**Comment by kozec** [comment]
Let me guess, Arch Linux? It looks like they upgraded GTK to version that's not backwards compatible.

But this particular problem is already fixed in #431

**Comment by codl** [comment]
oops yeah i didnt see the other issue

thanks!
